### ISSUE TITLE
🚨 Error: Cannot read property 'getAttribute' of undefined

### ISSUE THREAD

**Issue by ampprojectbot** [ampproject/amphtml#28645]
Details
---
**Error report:** [link](http://go/ampe/CI6Qxsae-v6n4AE)
**First seen:** Apr 9, 2020
**Frequency:** ~ 1,298/day

Stacktrace
---
<pre><code>Error: Cannot read property 'getAttribute' of undefined
    at doc (<a href="https://github.com/ampproject/amphtml/blob/2005150002001/src/dom.js#L704">src/dom.js:704</a>:4)
    at getUsableTarget_ (<a href="https://github.com/ampproject/amphtml/blob/2005150002001/extensions/amp-video-docking/0.1/amp-video-docking.js#L603">extensions/amp-video-docking/0.1/amp-video-docking.js:603</a>:11)
    at getTargetFor_ (<a href="https://github.com/ampproject/amphtml/blob/2005150002001/extensions/amp-video-docking/0.1/amp-video-docking.js#L652">extensions/amp-video-docking/0.1/amp-video-docking.js:652</a>:21)
    at <a href="https://github.com/ampproject/amphtml/blob/2005150002001/src/service/position-observer/position-observer-worker.js#L109">src/service/position-observer/position-observer-worker.js:109</a>:6
    at <a href="https://github.com/ampproject/amphtml/blob/2005150002001/src/service/position-observer/position-observer-worker.js#L144">src/service/position-observer/position-observer-worker.js:144</a>:6
</code></pre>

Notes
---
`@rsimha` modified `src/dom.js:702-705` in #21212 (May 16, 2019)
`@alanorozco` modified `extensions/amp-video-docking/0.1/amp-video-docking.js:603-603` in #26358 (Feb 24, 2020)
`@alanorozco` modified `extensions/amp-video-docking/0.1/amp-video-docking.js:652-655` in #19735 (Dec 13, 2018)
`@zhouyx` modified `src/service/position-observer/position-observer-worker.js:106-136` in #10813 (Sep 8, 2017)
`@zhouyx` modified `src/service/position-observer/position-observer-worker.js:144-144` in #10813 (Sep 8, 2017)

**Seen in:**
- 05-15 Beta (0002+1)
- 05-15 Control (0002+1)
- 05-15 Experimental (0002+1)
- 05-15 Nightly-Control (0002+1)
- 05-15 Stable (0002+1)
- 05-27 Nightly (2217)
- +5 more

**Possible assignees:** `@alanorozco`, `@zhouyx`

/cc @ampproject/release-on-duty

**Comment by nainar** [comment]
Pinging @alanorozco
